<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogCameraTest</class>
 <widget class="QDialog" name="DialogCameraTest">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1075</width>
    <height>635</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>摄像机功能测试</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 25 24pt &quot;思源黑体 CN Light&quot;;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_FaceDetect">
       <property name="minimumSize">
        <size>
         <width>800</width>
         <height>540</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>800</width>
         <height>540</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>6</number>
       </property>
       <item>
        <widget class="QLabel" name="label_FaceCapture">
         <property name="minimumSize">
          <size>
           <width>240</width>
           <height>300</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>240</width>
           <height>300</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(0, 0, 0);</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>18</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_TakePhoto">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>60</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>拍照</string>
           </property>
           <property name="icon">
            <iconset>
             <normalon>:/Image/Camera.png</normalon>
            </iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>48</width>
             <height>48</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>18</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>18</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButton_Oneself">
           <property name="minimumSize">
            <size>
             <width>170</width>
             <height>40</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QRadioButton::indicator { 
	width: 32px;
	height: 32px;
}
QRadioButton::indicator::unchecked {
	image: url(:/Image/RadioUnChecked.png);
}
QRadioButton::indicator::checked { 
	image: url(:/Image/RadioCheck.png);
}
</string>
           </property>
           <property name="text">
            <string>被监护人</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>18</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <spacer name="horizontalSpacer_6">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>18</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButton_Guardian">
           <property name="minimumSize">
            <size>
             <width>170</width>
             <height>40</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QRadioButton::indicator { 
	width: 32px;
	height: 32px;
}
QRadioButton::indicator::unchecked {
	image: url(:/Image/RadioUnChecked.png);
}
QRadioButton::indicator::checked { 
	image: url(:/Image/RadioCheck.png);
}
</string>
           </property>
           <property name="text">
            <string>监护人</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_7">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>18</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,0,0">
     <item>
      <widget class="QLabel" name="label_Info">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>60</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>提示:未成年人需要本人和监护人分别拍照</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_OK">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>60</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>60</height>
        </size>
       </property>
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
